aachen.py

```diff
@@ -63,11 +67,12 @@ def parse_url(url, today=False):
     regex = '\((?P<name>[\dA-Z]+)\)\s*(?P<value>[\w\s]+)'
     legend = buildLegend(legend, document.find(id='additives').text, regex=regex)
 
-    days = ('montag', 'dienstag', 'mittwoch', 'donnerstag', 'freitag',
-            'montagNaechste', 'dienstagNaechste', 'mittwochNaechste', 'donnerstagNaechste', 'freitagNaechste')
-    for day in days:
-        data = document.find('div', id=day)
-        headline = document.find('a', attrs={'data-anchor': '#' + day})
+    days = ('Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag',
+               'Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag')
```

Comment:
You actually have the exact same issue as the previous PR: https://github.com/mswart/openmensa-parsers/pull/60#discussion_r153779231

Reply:
This is not an issue at all. This is more of a design choice. I can change this if you want to, but it works this way. If you look at my other changes, a suffix is appended for the second half of the days in the array.

Reply:
Fair enough. However I would argue that suffixing them afterwards is really confusing. In the `days` tuple, it would be very obvious and simple. Maybe you could make this small change? :)